libraries/AP_HAL_ChibiOS/hwdef/KakuteF7/hwdef.dat

```diff
@@ -115,6 +115,8 @@ PB0  TIM3_CH3 TIM3 PWM(4) GPIO(53)
 PC9  TIM3_CH4 TIM3 PWM(5) GPIO(54)
 PA3  TIM5_CH4 TIM5 PWM(6) GPIO(55)
 
+PD12 TIM4_CH1 TIM4 PWM(7) GPIO(56) NODMA
```

Comment:
this is the LED strip in the schematic. We can use it as PWM, but please put in a comment about it being intended to be the LED strip. We will probably add support for these LED strips later.
Also note that to use it as a LED strip we will need to enable DMA on the channel.